packages/control-property-editor-common/.eslintignore

```diff
@@ -0,0 +1,3 @@
+dist
+test/data
```

Comment:
Please use `test/fixtures`. We decided to go for a consistent structure in #796 (I know, existing projects weren't converted yet)

Reply:
adapted. thanks